docs/en/platform-engineers/cloud-services.md

```diff
@@ -4,61 +4,102 @@ KubeVela can help you to provision and consume cloud resources with your apps ve
 
 ## Provision
 
-In this section, we will add a Alibaba Cloud's RDS service as a new workload type in KubeVela.
+In this section, we will add an Alibaba Cloud's RDS service as a new workload type in KubeVela.
 
 ### Step 1: Install and configure Crossplane
 
-We use [Crossplane](https://crossplane.io/) as the cloud resource operator for Kubernetes.
-This tutorial has been verified with Crossplane `version 0.14`.
+Refer to [Installation](https://github.com/crossplane/provider-alibaba/releases/tag/v0.5.0) to install Crossplane
+Alibaba provider v0.5.0.
+
+```
+$ kubectl crossplane install provider crossplane/provider-alibaba:v0.5.0
+
+$ kubectl create secret generic alibaba-account-creds -n crossplane-system --from-literal=accessKeyId=xxx --from-literal=accessKeySecret=yyy
+
+$ kubectl apply -f provider.yaml
+```
+
+`provider.yaml` is as below.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: crossplane-system
+
+---
+apiVersion: alibaba.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: default
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: alibaba-account-creds
+      key: credentials
+  region: cn-beijing
+```
+
+Note: We currently just use Crossplane Alibaba provider. But as we are about to use [Crossplane](https://crossplane.io/) as the 
+cloud resource operator for Kubernetes in the near future. So let's keep the following guide for future reference.
+
+>This tutorial has been verified with Crossplane `version 0.14`.
 Please follow the Crossplane [Documentation](https://crossplane.io/docs/), 
 especially the `Install & Configure` and `Compose Infrastructure` sections to configure
 Crossplane with your cloud account.
```

Comment:
```suggestion
```

These lines should be deleted because we are not using crossplane here

Reply:
Done